src/buttons.css

```diff
@@ -178,8 +174,8 @@ against .round class */
 
 .btn.btn--pill-hc {
   border-radius: 0 !important;
-  margin-left: 1px;
-  margin-right: 1px;
+  margin-left: 1px !important;
+  margin-right: 1px !important;
```

Comment:
I believe that all the `!important`s you've added to margin values are unnecessary.